lib/exsoda/writer.ex

```diff
@@ -45,7 +45,8 @@ defmodule Exsoda.Writer do
   end
 
   defmodule PrepareDraftForImport do
-    defstruct fourfour: nil
+    defstruct fourfour: nil,
+    nbe: nil
```

Comment:
lint: I think this is usually indented so you can see it's under the `defstruct`